app/src/main/java/ch/epfl/sweng/swenggolf/offer/ShowOfferActivity.java

```diff
@@ -425,4 +444,25 @@ public void onCancelled(DbError error) {
 
     }
 
+    private void hideReactButton() {
+        Button reactButton = inflated.findViewById(R.id.react_button);
+        reactButton.setVisibility(View.GONE);
```

Comment:
Is the button still clickable although it is not visible ? Normally it shouldn't but just to be sure.